hack/refresh-examples.sh

```diff
@@ -31,53 +35,57 @@ naYJRuLprwIv6FDhZ5yFjYUEtsmoNcW7rx2KM6FOXGsCX3BNc7qhHELT+g==
 INPUT_FILE="${ROOT_DIR}/acceptance/samples/policy-input-golden-container.json"
 TRUSTED_TASKS_FILE="${ROOT_DIR}/example/data/trusted_tekton_tasks.yml"
 
-trap 'rm "${TRUSTED_TASKS_FILE}-update"' EXIT
+trap 'rm -f "${TRUSTED_TASKS_FILE}-update"' EXIT
 
-go run -modfile "${ROOT_DIR}/go.mod" github.com/enterprise-contract/ec-cli validate image \
-    --images <(cat << EOF
-{
+DIGEST="$(skopeo inspect --no-tags "docker://${IMAGE}" --format '{{index .Digest}}')"
+ORIGINAL_IMAGE_REF="${ORIGINAL_IMAGE_REPO}@${DIGEST}"
+REVISION="$(skopeo inspect --no-tags "docker://${ORIGINAL_IMAGE_REF}" --format '{{index .Labels "vcs-ref"}}')"
```

Comment:
Ooh, we should use this technique in hack/simple-demo.sh instead of trying to get it from api.github.com.